### PR TITLE
Prevent "class not available" error (Adaption towards Contao 5 Step 002)

### PR DESCRIPTION
**Pull request by Ernestopheles**
Next adaption towards Contao 5.

See useCases tested with cypress: [tests](https://github.com/Ernestopheles/isotopeDemo/tree/cypress/cypress)
Contao 4.13.31, Isotope 2.9 and PHP 8.1

### CODE REVIEW

**Comment by Ernestopheles** [comment]
First adaption (002) towards Contao 5.

Take Template from Isotope\Template instead of \Template to prevent "class not available" error in ProductCollection::addToTemplate.

This error occured during cache warm-up under Contao 5.2, PHP 8.1 as class \Template was not available. 

See useCases tested with cypress: [tests](https://github.com/Ernestopheles/isotopeDemo/tree/cypress/cypress)
Contao 4.13.31, Isotope 2.9 and PHP 8.1

File: system/modules/isotope/library/Isotope/Interfaces/IsotopeProductCollection.php [hunk 1]
```diff
@@ -16,6 +16,7 @@
 use Isotope\Model\Config;
 use Isotope\Model\ProductCollectionItem;
 use Isotope\Model\ProductCollectionSurcharge;
+use Isotope\Template;
```
Comment (aschempp) [inline system/modules/isotope/library/Isotope/Interfaces/IsotopeProductCollection.php]: This should be `Contao\Template`, it does not have to be an Isotope-specific one.

**Review by aschempp — CHANGES_REQUESTED** [review]
Good find! And a perfect example of something we can & should & will fix in the 2.9 version since that affects Contao 4 as well. Just one minor change request regarding the namespace.

File: system/modules/isotope/library/Isotope/Interfaces/IsotopeProductCollection.php (hunk 1 above)
Comment (Ernestopheles) [reply system/modules/isotope/library/Isotope/Interfaces/IsotopeProductCollection.php]: Done

**Review by aschempp — APPROVED** [review]

**Comment by aschempp** [comment]
Thank you @Ernestopheles

**Pull request merged**